test/factories/coral_cover.jl

```diff
@@ -0,0 +1,60 @@
+using ADRIA: YAXArray, DataCube, ZeroDataCube
+
+"""Raw model coral cover results, proportional to k-area."""
+function coral_cover(
```

Comment:
I just realized that the one I'm calling `coral_cover` is the one that sums up to one and the one I'm calling `full_coral_cover` is the one that doesn't. I'll fix that.